hw/top_englishbreakfast/data/top_englishbreakfast.hjson

```diff
@@ -86,6 +86,7 @@
 
     groups: [
       // the powerup group is used exclusively by clk/pwr/rstmgr/pinmux
```

Comment:
Move this comment so it stays with "powerup".

Reply:
fixed!